android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -977,6 +986,48 @@ protected ViewGroup getRootView() {
     private OnScrollDispatchHelper mOnScrollDispatchHelper;
     protected boolean hasScrollEvent = false;
 
+    @Override
+    public InputConnection onCreateInputConnection(EditorInfo editorInfo) {
+        WebView webView = this;
+        final InputConnection ic = super.onCreateInputConnection(editorInfo);
+        EditorInfoCompat.setContentMimeTypes(editorInfo,
+                new String [] {"image/*", "image/png", "image/gif", "image/jpeg", "video/mp4"});
```

Comment:
Do I explicitly list all the image types supported, or just `"image/*"`? I think I took this line from the Android docs, but so far only `"image/gif`" has ever shown up when using the gif picker from the keyboard.